src/script/descriptor.cpp

```diff
@@ -1509,6 +1509,10 @@ std::vector<std::unique_ptr<PubkeyProvider>> ParsePubkeyInner(uint32_t key_exp_i
         error = "No key provided";
         return {};
     }
+    if (IsSpace(str.front()) || IsSpace(str.back())) {
```

Comment:
*(comment about the commit, not this line)*

nit: in commit e91a9cf9d2ca50ee31ed36a975d2e01faa666c92 you state:
> Due to Base58, pubkeys with whitespace at the beginning
end/or at the end are successfully parsed.

But pubkeys are never base58 encoded.

Reply:
I meant due to `DecodeExt{Pub}Key` which calls `DecodeBase58Check`.